playbook.yml

```diff
@@ -123,16 +129,10 @@
         path: C:\tmp\install\pgadmin3-1.18.1\pgadmin3.msi
         state: present
 
-    - name: Download Notepad++
-      win_shell: |
-        $url = "https://notepad-plus-plus.org/repository/7.x/7.5.5/npp.7.5.5.Installer.exe"
-        $output = "C:\tmp\install\npp.7.5.5.Installer.exe"
-        $wc = New-Object System.Net.WebClient
-        $wc.DownloadFile($url, $output)
-
     - name: Install Notepad++
-      win_shell: |
-        C:\tmp\install\npp.7.5.5.Installer.exe /S
+      win_chocolatey:
```

Comment:
Can we get away from notepadplusplus and install `choco install atom --pre` Atom instead/along with Notepad?